stellarphot/io/tests/test_tess_submission.py

```diff
@@ -3,7 +3,7 @@
 
 import pytest
 from astropy.coordinates import SkyCoord
-from requests import ConnectionError
+from requests import ConnectionError, ReadTimeout
```

Comment:
I'm guessing this has to do with allowing timeout of TESS access attempt, not the name field restrictions.

Reply:
Correct...I wanted to make the lights green